### ISSUE TITLE
(tab)open completions should show matching searchurls

### ISSUE THREAD

**Issue by tpikonen** [tridactyl/tridactyl#1010]
Completions for open, tabopen etc. should show the matching keywords from searchurls. Vimperator had these prioritized first in the completions list.

For example, when typing 'tabopen wiki' the first completion match should read something like 'Search wikipedia', and pressing tab should complete 'tabopen wikipedia ' to the command line.

**Comment by glacambre** [comment]
Hi, thank you for opening. This is planned, but we have to rewrite the whole commandline first as it is currently unable to display more than one completion category.

Related pull request: https://github.com/tridactyl/tridactyl/pull/932

**Comment by matt-snider** [comment]
Also wondering if there's some way to do this or if there's a way to limit the completions to keys in `searchurls`. I use quite a lot of them and don't always remember what key was pressed, it would be nice to have something like `open search` or even a separate command `search` where the completions would be limited in this way.

**Comment by rddunphy** [comment]
It would also be nice if there was visual feedback when a keyword has been entered, to let the user know that their input will be interpreted as a search. This would mean if I'm searching a keyword I don't use very often, I don't have to wait for the page to load to see if I remembered it correctly - also, if a search term is actually a keyword I'd forgotten about, I'm forewarned that it won't take me where I'm expecting. For example, I have `i` bound to Google Image search. If I type in a phrase that happens to start with the word I ("i spy", say) this backfires. If the prompt changed to "Search Google Images", I'd realise my mistake. (Yes, I'm aware I've just spent five minutes asking for a feature that might one day save me two seconds.)
